.github/labeler.yml

```diff
@@ -0,0 +1,35 @@
+# Add a labeler to any changes within folders or any subfolders
+
+# Seismological software: software name is used as the labeler
+cap:
+  - any: ['source/cap/**/*']
```

Comment:
If I understand it correctly, the above line can be simplified to 
```
cap: source/cap/**/*
```

Perhaps we should change it to this more compact way?

Reply:
done